pkg/pipelines/tekton/tasks.go

```diff
@@ -5,7 +5,24 @@ import (
 	"strings"
 )
 
-var DeployerImage = "ghcr.io/knative/func-utils:v2"
+var (
+	FuncUtilImage = "ghcr.io/knative/func-utils:v2"
```

Comment:
where is the code for this container?

Reply:
wdym?
Dockerfile is here: https://github.com/knative/func/blob/main/Dockerfile.utils

Reply:
This is not a new image.